docs/API.md

```diff
@@ -696,7 +696,8 @@ Then you will receive such notifications when a transaction is impacting the `de
         "value": 100000000
       }
     ],
-    "cryptoCode": "BTC"
+    "cryptoCode": "BTC",
+    "replacing": ["25d6bc1b2812670550aca8b2984670203b5ebf00e75f9b2bbf1940c3fa27841e", "81a20eb55ec16b92c65d4e142278fd521caa9e5dcad9d941c8e256dbd917ae84"]
```

Comment:
Is this only for events over websocket or does it apply to long polling as well? If yes, then this is *exactly* what I need. I use long polling.

Reply:
yes it does

Reply:
Hi i have no idea of what im doing or what your talking about  if u can only help me ou t ill really appreciate it i dont know if this is real or is fraud